run/preprod/docker/.env

```
@@ -0,0 +1 @@
+NETWORK="preprod"
```

```suggestion
NETWORK="preprod"

```
(newline at end of file)